stdlib/jdbc/src/main/ballerina/src/java.jdbc/client_endpoint.bal

```diff
@@ -24,7 +24,7 @@ public type Client client object {
 
     # Gets called when the JDBC client is instantiated.
     public function __init(ClientEndpointConfig c) {
-        self.jdbcClient = createClient(c, getGlobalPoolConfigContainer().getGlobalPoolConfig());
+        self.jdbcClient = createClient(c, globalPoolConfigContainer);
```

Comment:
Shouldn't this be ` globalPoolConfigContainer.getGlobalPoolConfig()`?

Reply:
Yes, it should. Fixed.